src/__tests__/06.extra-2.js

```diff
@@ -2,7 +2,7 @@ import React from 'react'
 import {render, screen} from '@testing-library/react'
 import userEvent from '@testing-library/user-event'
 import App from '../final/06.extra-2'
-// import App from '../exercise/06'
+// import App from '../exercise/06.extra-2'
```

Comment:
People who are working through the workshop won't create these files. They'll just keep working in the exercise file.